saleor/tests/fixtures.py

```diff
@@ -3410,9 +3412,14 @@ def order_line_with_one_allocation(
     Allocation.objects.create(
         order_line=order_line, stock=stocks[0], quantity_allocated=1
     )
-    stock = stocks[0]
-    stock.quantity_allocated = 1
-    stock.save(update_fields=["quantity_allocated"])
+    stocks_to_update = []
+    for mismatched_stock in Stock.objects.annotate(
+        allocations_allocated=Coalesce(Sum("allocations__quantity_allocated"), 0)
+    ).exclude(quantity_allocated=F("allocations_allocated")):
+        mismatched_stock.quantity_allocated = mismatched_stock.allocations_allocated
+        stocks_to_update.append(mismatched_stock)
+    if stocks_to_update:
+        Stock.objects.bulk_update(stocks_to_update, ["quantity_allocated"])
```

Comment:
Do we really need to do smth like thin in the fixture? Cannot we just create new stocks with correct `quantity_allocated` values?